tests/foreman/cli/test_subnet.py

```diff
@@ -89,6 +90,26 @@ def test_create_subnet_with_domain(self):
             self.fail(err)
         self.assertIn(domain['name'], subnet['domains'])
 
+    def test_create_subnet_with_some_domains(self):
```

Comment:
'some' doesn't seems to look good here. 'some' may have different meanings. You may try something like 'multiple' or so.